service/history/replication/ack_manager.go

```diff
@@ -453,6 +461,17 @@ func (p *ackMgrImpl) ConvertTask(
 			task,
 			p.workflowCache,
 		)
+	default:
+		return nil, errUnknownReplicationTask
+	}
+}
+
+func (p *ackMgrImpl) ConvertTaskByCluster(
```

Comment:
nit: do you want to move this change into next PR as well?

Reply:
It's would be simple to keep in this PR because the subsequent functions' signature is updated with cache and targetClusterID.